README.md

```diff
@@ -95,6 +95,8 @@ Install this project with:
 
 To run the project, execute the following command:
 
+Link to [React App](https://github.com/Shakir-Hussain12/hello-react-frontend)
+
 ```sh
   rails s
   for React: bin/dev
```

Comment:
- [ ] You need to update your readme file to add this project's related data, you have some lines which refer to webpack project => for example this one: ` git clone https://github.com/Shakir-Hussain12/hello_webpack.git`
- [ ] And instead of  `cd <proj_name>`, please specify the folder name